spinalcordtoolbox/scripts/sct_detect_pmj.py

```diff
@@ -256,6 +262,67 @@ def ifolder2tmp(self):
         self.curdir = os.getcwd()
         os.chdir(self.tmp_dir)  # go to tmp directory
 
+    def compute_cross_corr_3d(self, xrange=list(range(-10, 10)), xshift=10, yshift=10, zshift=10):  # Range of 20 is bit extreme?
```

Comment:
Comment to remove before the merge. I don't find it extreme. The only problem is that it is dependent on the voxel resolution. So, if someone arrives with an image with a resolution of 0.1x0.1x0.1mm, then 20 might not be enough. So ideally you would adjust this parameter to be in a physical dimension (mm).

Reply:
I agree, I'll change it to physical dimension!

Reply:
done in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3468/commits/0ef6d4fc2789f4003dc9c3f197536baf0f125102